systers_portal/meetup/views.py

```diff
@@ -0,0 +1,17 @@
+from django.views.generic.detail import DetailView
+
+from meetup.models import MeetupLocation
+
+
+class MeetupLocationDetailView(DetailView):
+    """Meetup Location page view, show only details that user requested"""
+    model = MeetupLocation
+
+    def get_context_data(self, **kwargs):
+        """Add slugs to context and set template_name as per page_slug value"""
+        context = super(MeetupLocationDetailView,
+                        self).get_context_data(**kwargs)
+        context['slug'] = self.kwargs['slug']
+        context['page_slug'] = self.kwargs['page_slug']
+        self.template_name = 'meetup/{0}.html'.format(self.kwargs['page_slug'])
```

Comment:
Templates are decided as per second slug(page_slug) which have values - about, upcoming, past, organizers, members. These values will come from base.html and meetup_sidebar.html inside template/meetup folder.

Since I have designed template separately for each page_slug value with one base.html template, it need to "dynamically" decide which template will be used as per the value of page_slug value.